wgsl/index.bs

```diff
@@ -3339,7 +3353,7 @@ If a reference or pointer access is out of bounds, an
 [=Load Rule|Loads=] from an invalid reference return one of:
     * when the [=originating variable=] is a [=uniform buffer=] or a [=storage buffer=],
         the value from any [=memory locations|memory location(s)=]
-        of the [[WebGPU#buffers|WebGPU buffer]] bound to the originating variable
+        of the {{GPUBuffer}} bound to the originating variable
```

Comment:
nit: I'd prefer this to say   "WebGPU {{GPUBuffer}}" so readers don't go looking in this doc.